notes.js

```diff
@@ -4,17 +4,68 @@ app.use(express.json());
 const db = require('./db/connection');
 
 
-const { serverError, handleCustomErrors } = require('./errors');
+const { serverError, handleCustomErrors, psqlError } = require('./errors');
+const { fetchArticleId } = require('./models/GET-models');
 
 
+const getArticles = ((req, res, next) => {
+
+    console.log(req.query, "controller");
+    const { sortby, order, topic } = req.query;
+    console.log(req.query.hasOwnProperty('comment_count'));
+
+    const comment_count = req.query.hasOwnProperty('comment_count') ? 1 : 0;
```

Comment:
We shouldn't be providing a query of comment_count here
We could have a `?sort_by=comment_count`
But all articles should have a comment_count by default